react/src/components/RegularSeasonTable.js

```diff
@@ -85,19 +86,18 @@ const RegularSeasonTable = ({ column, data, isLoaded, direction, handleSort }) =
               data,
               (
                 {
+                  isChamp,
```

Comment:
Does order matter here? I have to admit I just kept these properties in the same order the came out of the GetData() call because I wasn't sure if they were ordered or based on the variable name.

Reply:
Nope it doesn't matter, because it's an object. I tend to alphabetize but I forget why I moved `isChamp` to the top lol.